core-web/libs/data-access/src/lib/dot-copy-content/dot-copy-content.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Observable } from 'rxjs';
+
+import { Injectable } from '@angular/core';
+
+import { pluck, shareReplay } from 'rxjs/operators';
+
+import { CoreWebService, DotRequestOptionsArgs } from '@dotcms/dotcms-js';
+import { DotCopyContent, DotCMSContentlet } from '@dotcms/dotcms-models';
+
+export const DEFAULT_PERSONALIZATION = 'dot:default';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class DotCopyContentService {
+    constructor(private readonly coreWebService: CoreWebService) {}
+
+    /**
+     *
```

Comment:
Add desc.

Reply:
I'll do it!

Reply:
Remove the extra line, not sure if the parse will parse like thie.